### DISCUSSION TITLE
Add a "buy me a coffee" link or similar to the Repo.

### DISCUSSION THREAD

**Discussion by Dark1886** [OMOTE-Community/OMOTE-Firmware#27] [Ideas]
You should allow a "buy me a coffee" link to your repo. You have already spent quite a bit of time working on this project and more time will be spent between you and others. Adding a coffee link could be a way for some to thank you for the efforts or fund future board revision purchases that could be sent to common contributors.

**Comment by dekiesel** [comment]
That's a great idea!
